server/gql/models/books/index.js

```diff
@@ -10,15 +10,16 @@ import { sequelizedWhere } from '@server/database/dbUtils';
 import { insertBook, updateBook } from '@server/daos/books';
 import { insertAuthorsBooks, updateAuthorsBooksForBooks } from '@server/daos/authorsBooks';
 import { authorsBookFieldsMutation } from '../authorsBooks';
+import { LanguageConnection } from '../languages';
```

Comment:
instead of relative imports let's use "@gql/models/languages" there is no performance difference, just reads better

Reply:
okay I'll implement that